src/redux/notificationsSlice.js

```diff
@@ -36,9 +55,21 @@ export const notificationsSlice = createSlice({
       state.error = action.error.message;
       state.pending = false;
     },
+    [updateNotification.pending]: (state) => {
+      state.pending = true;
+      state.error = false;
+    },
+    [updateNotification.fulfilled]: (state) => {
+      state.pending = false;
+    },
+    [updateNotification.rejected]: (state, action) => {
+      state.pending = false;
+      state.error = action.error.message;
+    },
```

Comment:
jel se update radi samo za to kada su otvorene notifikacije?

fali ti neko postavljanje tih notifikacije onda, sad samo menjas pending flag u ovom zahtevu